packages/terra-clinical-data-grid/src/DataGrid.jsx

```diff
@@ -1084,7 +1094,12 @@ class DataGrid extends React.Component {
         >
           <div
             className={cx('vertical-overflow-container')}
-            ref={this.setVerticalOverflowContainerRef}
+            ref={(ref) => {
+              this.setVerticalOverflowContainerRef(ref);
+              if (verticalOverflowContainerRefCallback) {
```

Comment:
This logic should be added to [`this.setVerticalOverflowContainerRef`](https://github.com/cerner/terra-clinical/blob/b069773ed557b1c21ab850266334bffe3dfcc245/packages/terra-clinical-data-grid/src/DataGrid.jsx#L573)
```diff
  setVerticalOverflowContainerRef(ref) {
    this.verticalOverflowContainerRef = ref;
+    if (this.props.verticalOverflowContainerRefCallback) {
+       verticalOverflowContainerRefCallback(ref)
+    }
  }

```

Reply:
Updated here https://github.com/cerner/terra-clinical/pull/585/commits/9de4536895fbfa7f35601a8b54b810108df7b7da.